account_payment_partner/models/account_move.py

```diff
@@ -79,6 +79,13 @@ def _compute_payment_mode(self):
                             partner.supplier_payment_mode_id.refund_payment_mode_id
                         )
 
+    # Force compute because onachange partner_id don't execute _compute_invoice_partner_bank
+    @api.onchange("partner_id")
+    def _onchange_partner_id(self):
```

Comment:
This should be put as docstring instead:
```suggestion
    @api.onchange("partner_id")
    def _onchange_partner_id(self):
        """Force compute because onachange partner_id don't execute 
        _compute_invoice_partner_bank.
        """
```